jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java

```diff
@@ -51,4 +52,52 @@
     public static Checkbox configurableResultCheckbox;
 
     public static SlideToggleSection slideToggleSection;
+
+    @UI("#mat-input-12")
+    public static TextField foodBasicInput;
```

Comment:
Please create a separate sections for all inputs. We should not put all the small elements to the same AngularPage, this way we will end up with huge amount of elements defined in one class.